### DISCUSSION TITLE
Is it possible to wait exactly until an item loads?

### DISCUSSION THREAD

**Discussion by mvkvc** [keathley/wallaby#727] [Q&A] [answered]
While a page is loading, is it possible to run commands exactly after an item on the page loads? Ex. wait on specific image loading and then take a screenshot, the current way I am trying it is to do a manual `Process.sleep`.

Thank you!

**Comment by mhanberg** [comment, marked as answer]
Using the find function will wait for it to appear, and you can pass it a callback. You can probably utilize that mechanism.

**Reply by mvkvc** [reply]
Thank you!
